### ISSUE TITLE
How to use TPOT in the text domain?

### ISSUE THREAD

**Issue by ben0it8** [rhiever/tpot#544]
Hello,

My question is if it's possible to use TPOT in the text domain for classification task?
Given a labeled corpus (eg. label - document pairs) I'd like to perform classification to infer the label of an unseen piece of document.

Thanks,
Oliver

**Comment by weixuanfu** [comment]
I think this issue is related to #507. We are working on a configurable grammar in #523 to add the support for text classification. For now, you may try to transform text to numeric matrix using [CountVectorizer](http://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.CountVectorizer.html), [TFIDFVectorizer](http://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfVectorizer.html) and [HashingVectorizer](http://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.HashingVectorizer.html) before using TPOTClassifier for your problem.
